integration-tests/contract-failing-noop.spec.ts

```diff
@@ -1,34 +1,30 @@
 import { InMemorySigner } from "@taquito/signer";
-import { CONFIGS, defaultSecretKey, isSandbox } from "./config";
-import { OpKind, Protocols, TezosToolkit } from "@taquito/taquito";
+import { CONFIGS, defaultSecretKey } from "./config";
+import { OpKind, TezosToolkit } from "@taquito/taquito";
 import { verifySignature } from "@taquito/utils";
 
-CONFIGS().forEach(({ rpc, setup, protocol}) => {
-
+CONFIGS().forEach(({ setup }) => {
+  let rpc = 'https://mainnet-archive.ecadinfra.com/'
```

Comment:
Could we add this into the new config that Arvid help build? Seems like we can probably move away from using direct string variables for urls, unless there's something preventing us from doing so.

Reply:
What arvid have refactored is for testnets, this is the set up. Think we don't mind querying rpc for data from mainnet but we don't want to run test with configSigner or secretKey on mainnet.
<img width="993" alt="image" src="https://github.com/ecadlabs/taquito/assets/106410553/19d6ea2e-974a-4b11-a18b-2b5bada754bc">